### PR TITLE
Fix for IE 11-compatible Babel transpilation

### PR DESCRIPTION
**Pull request by opyh**
This cleans up browserslist / `babel/preset-env` configuration and fixes IE11 by moving `babel-plugin-ttag` out of the way when Next.js / Webpack / Babel transpiles our code.

Here is a more detailed description of the issue: https://github.com/ttag-org/babel-plugin-ttag/issues/138

For fun, it includes little changes to remove deprecation warnings in Next 9.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,15 +1,34 @@
 {
-  "presets": ["next/babel"], // https://goo.gl/aqbWJY
+  "presets": [[
+    "next/babel",
+    {
+      "preset-env": {
+        "useBuiltIns": "entry",
+        "corejs": 3,
```
Comment (opyh) [inline .babelrc]: Newer than the default. Default would be 2, also this line is necessary to get rid of a warning.

File: .babelrc [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+        "debug": true
+      },
+      "transform-runtime": {
+        "corejs": 3,
+        "useESModules": false
```
Comment (opyh) [inline .babelrc]: Without this, the server code would crash on start.

File: .babelrc [hunk 3]
```diff
@@ -1,15 +1,34 @@
...
+  ]], // https://goo.gl/aqbWJY
   "plugins": [
     "transform-flow-strip-types",
-    [
-      "ttag",
-      {
+    "babel-plugin-styled-components"
+  ],
+  "env": {
+    "ttag": {
```
Comment (opyh) [inline .babelrc]: This is the configuration used when running Babel with `BABEL_ENV=ttag`.

File: package.json [hunk 4]
```diff
@@ -143,9 +147,9 @@
     "version": "echo TODO: Add version syncing to bundle configurations for native apps + git commit -am 'Version bump' to package.json",
     "flow": "flow",
     "server": "server",
-    "push-translations": "npm run clear-babel-cache && npm run build && tx push -s",
+    "push-translations": "npm run clear-babel-cache && (BABEL_ENV=ttag npm run build) && tx push -s",
     "pull-translations": "tx pull -af",
-    "retranslate": "npm run build && npm run pull-translations && NEW_SOURCE_PO='./src/static/i18n/en_US.txt' retranslate/run.sh",
```
Comment (opyh) [inline package.json]: `static` is deprecated

File: package.json [hunk 5]
```diff
@@ -184,7 +188,8 @@
     "Opera >= 53",
     "iOS >= 10",
     "last 1 version",
-    "> 1%"
+    "> 0.25%",
+    "not dead"
```
Comment (opyh) [inline package.json]: Support a few more browsers because why not :)

File: src/pages/_app.js [hunk 6]
```diff
@@ -313,72 +313,70 @@ export default class App extends BaseApp {
     };
 
     return (
-      <Container>
```
Comment (opyh) [inline src/pages/_app.js]: Next.js 9 doesn't need this component wrapper anymore. Everything below this line in this file is just Prettier's re-indentation, feel free to ignore :)

File: .babelrc (hunk 3 above)
Comment (mutaphysis) [reply .babelrc]: did you test the pot file extraction with these changes?

Comment (opyh) [reply .babelrc]: I didn't look in detail at the `.pot` file, but it was generated correctly.

**Pull request merged**